### DISCUSSION TITLE
Add the ics to PHP Mailer

### DISCUSSION THREAD

**Discussion by GutyECUADOR** [markuspoerschke/iCal#267] [Q&A]
Hi guys any advise to add the ICS file like attachment with PHPMailer? I try $mailer->addAttachment($this->generateISC(), 'event.ics') But dont work;

**Comment by soderlind** [comment]
Long time since you asked, but if somebody else is wondering. I used the following (in WordPress): https://gist.github.com/thomasfw/5df1a041fd8f9c939ef9d88d887ce023/

You can look at the code in the gist to see how it's done and borrow part of the code if you're using PHP Mailer stand alone or in another framework.
